<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>在此处添加角色与环节</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>当前角色与环节</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_current_player"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>角色(Player)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_player"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>环节(Section)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_section"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_add_player">
         <property name="text">
          <string>添加角色</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;File)</string>
    </property>
    <addaction name="action_export"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_export">
   <property name="text">
    <string>导出(&amp;Export)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
